### PR TITLE
Add support for tsflags=nodocs

### PR DESCRIPTION
**Pull request by AZaugg**
Allow tdnf to honor the nodocs tsflags.

### CODE REVIEW

File: client/rpmtrans.c [hunk 1]
```diff
@@ -115,6 +115,11 @@ TDNFRpmCreateTS(
         {
             pTS->nTransFlags |= RPMTRANS_FLAG_NOSCRIPTS;
         }
+
+        if (!strcasecmp(pSetOpt->pszOptValue, "nodocs"))
+        {
+            pTS->nTransFlags |= rpm.RPMTRANS_FLAG_NODOCS;
```
Comment (oliverkurth) [inline client/rpmtrans.c]: Please fix this:
```
/build/client/rpmtrans.c: In function 'TDNFRpmCreateTS':
/build/client/rpmtrans.c:121:33: error: 'rpm' undeclared (first use in this function); did you mean 'rpmop'?
  121 |             pTS->nTransFlags |= rpm.RPMTRANS_FLAG_NODOCS;
      |                                 ^~~
      |                                 rpmop
/build/client/rpmtrans.c:121:33: note: each undeclared identifier is reported only once for each function it appears in

Should probably just be `RPMTRANS_FLAG_NODOCS`.

**Review by oliverkurth — CHANGES_REQUESTED** [review]
Thanks for the contribution!

Please see the section "Testing Changes" in https://github.com/vmware/tdnf/blob/dev/CONTRIBUTING.md . That would have caught that compile error.

File: client/rpmtrans.c (hunk 1 above)
Comment (AZaugg) [reply client/rpmtrans.c]: Apologies, I had a copy-paste error and I forgot to git add;commit my typo changes

**Review by oliverkurth — APPROVED** [review]

**Comment by oliverkurth** [comment]
Can you please update the comment in line104/105 too? It's no longer true:
```
    /* parse transaction flags - so far only tsflags=noscripts is
       supported. */
```

**Comment by oliverkurth** [comment]
Thanks for the contribution!

**Pull request merged**